package-config.cmake

```diff
@@ -81,77 +96,91 @@ macro(ADD_PROJECT_DEPENDENCY)
     ""
     ${ARGN}
   )
-  if(PARSED_ARGN_PKG_CONFIG_REQUIRES)
+
+  list(GET PARSED_ARGN_UNPARSED_ARGUMENTS 0 _package_name)
+  if(NOT ${_package_name} IN_LIST PROJECT_PACKAGES_IN_WORKSPACE)
+    find_package(${PARSED_ARGN_UNPARSED_ARGUMENTS})
```

Comment:
That `find_package` is now before the `set(CMAKE_MODULE_PATH …)`, which is breaking the feature

Reply:
Nice catch. This should be fixed now.